src/NServiceBus.AcceptanceTesting/ContextAppender.cs

```diff
@@ -7,17 +7,11 @@
     // This class is written under the assumption that acceptance tests are executed sequentially.
```

Comment:
I believe this comment is no longer true with these changes, correct? Should be able to remove this.